client/landing/gutenboarding/index.tsx

```diff
@@ -30,7 +32,14 @@ window.AppBoot = () => {
 
 		ReactDom.render(
 			<BrowserRouter basename="gutenboarding">
-				<Gutenboard />
+				<Switch>
+					<Route exact path={ path }>
+						<Gutenboard />
+					</Route>
+					<Route>
+						<Redirect to={ makePath( Step.IntentGathering ) } />
```

Comment:
Do we need this here though? Could potentially move the fallback into the onboarding block's routing, no? :thinking: